packages/spor-react/src/loader/Skeleton.tsx

```diff
@@ -1,42 +1,71 @@
 "use client";
+import { skeletonRecipe } from "@/theme/recipes/skeleton";
 import type {
   SkeletonProps as ChakraSkeletonProps,
   CircleProps,
+  RecipeVariantProps,
+} from "@chakra-ui/react";
+import {
+  Skeleton as ChakraSkeleton,
+  Circle,
+  Stack,
+  useRecipe,
 } from "@chakra-ui/react";
-import { Skeleton as ChakraSkeleton, Circle, Stack } from "@chakra-ui/react";
 import * as React from "react";
 import { forwardRef } from "react";
 
-export interface SkeletonCircleProps extends ChakraSkeletonProps {
-  size?: CircleProps["size"];
-}
+type SkeletonVariantProps = RecipeVariantProps<typeof skeletonRecipe>;
+
+export type SkeletonCircleProps = ChakraSkeletonProps &
+  SkeletonVariantProps & {
+    size?: CircleProps["size"];
+  };
 
 export const SkeletonCircle = React.forwardRef<
   HTMLDivElement,
   SkeletonCircleProps
 >(function SkeletonCircle(props, ref) {
-  const { size, ...rest } = props;
+  const recipe = useRecipe({ recipe: skeletonRecipe });
+
+  const [recipeProps, restProps] = recipe.splitVariantProps({
+    loading: true,
+    variant: "pulse",
+    ...props,
+  });
+
+  const styles = recipe(recipeProps);
+  const { size, ...rest } = restProps;
+
   return (
     <Circle size={size} asChild ref={ref}>
-      <ChakraSkeleton {...rest} />
+      <ChakraSkeleton {...rest} css={styles} />
```

Comment:
Enig der, kanskje dette vil fungere bedre

```suggestion
      <ChakraSkeleton {...rest} css={{ ...styles, ...props.css }} />
```